src/App.test.js

```diff
@@ -1,14 +1,15 @@
 import React from 'react';
 import App from './App';
-import { shallow } from 'enzyme';
+
+import { render, screen } from '@testing-library/react';
 
 describe('React Step 1 Tests', () => {
   it('renders without crashing', () => {
-    shallow(<App />);
+    render(<App />);
   });
 
-  const wrapper = shallow(<App />);
-  it('contains a TutorialHeader', () => {
-    expect(wrapper.find('TutorialHeader').length).toBe(1);
+  it('contains Carbon Tutorial heading', () => {
+    render(<App />);
+    expect(screen.getByText('Carbon Tutorial')).toBeInTheDocument();
   });
 });
```

Comment:
I'm getting an error when trying to run this on the step 2 branch 🤔
<img width="1243" alt="image" src="https://github.com/carbon-design-system/carbon-tutorial/assets/40550942/fbae9701-fc95-40a3-bd5c-6135a2f92b10">

Reply:
Should be fixed now 🙏